nightshift/datastore_transactions.py

```diff
@@ -42,6 +42,29 @@ def init_db() -> None:
     session.close()
 
 
+def add_resource(session: Session, resource: Resource) -> Optional[Resource]:
+    """
+    Adds Resource record to db.
+
+    Args:
+        session:                `sqlalchemy.Session` instance
+        resource:               `datastore.Resource` object
+
+    Returns:
+        `Resource` instance updated with `nid`
```

Comment:
I think a db `flush` would need to be called to populate the `nid` field in this method, but perhaps that happens where this is invoked?

Reply:
I think it make sense to add it here. I initially forgot about this part but will add it to PR #11 .